doc/GIT.md

```diff
@@ -18,20 +29,34 @@ git config --global merge.ff false
 git config merge.ff false
```

Comment:
Good improvement to change that to the repository only as it totally depends on the strategy of the upstream repository. In this merging section I would also add `git config pull.rebase true` which will automatically rebase when pulling which makes it unnecessary calling it manually.